### PR TITLE
Docs: Comprehensive renderer tutorial

### PR DESCRIPTION
**Pull request by TorsteinHonsi**
Based on website traffic, content on the SVG Renderer is popular. Yet we had no comprehensive tutorial explaining how to best use it.

### CODE REVIEW

**Review by raf18seb — APPROVED** [review]
Nice and clean! Nothing to add 👏

File: docs/advanced-chart-features/renderer.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+Note that we haven't supplied the actual [path definition](https://api.highcharts.com/class-reference/Highcharts.SVGAttributes#d) yet, and we have not
+set the x and y positions of the text. What we do, is to register the elements
+directly on the Chart instance (`this`), so that subsequent render events will
+pick them up and reuse them instead of creating new ones. Also note that we
+define a flag, `isNew`, that we will use below.
+
+After doing some math to determine exactly where to put the elements, we do the
+actual placement:
+
+```
```
Comment (KacperMadej) [inline docs/advanced-chart-features/renderer.md:127]: Why this code block is not marked as JS with **```js** ?

File: ts/Core/Renderer/SVG/SVGElement.ts [hunk 2]
```diff
@@ -122,6 +122,10 @@ interface TextPathObject {
  * {@link Highcharts.SVGRenderer#g|g}
  * and more.
  *
+ * See [How to use the SVG Renderer](
+ * https://www.highcharts.com/docs/advanced-chart-features/boost-module) for a
```
Comment (KacperMadej) [inline ts/Core/Renderer/SVG/SVGElement.ts:126]: Shouldn't this be linked to the renderer docs?

File: ts/Core/Renderer/SVG/SVGRenderer.ts [hunk 3]
```diff
@@ -101,6 +101,10 @@ let hasInternalReferenceBug: (boolean|undefined);
  * An existing chart's renderer can be accessed through {@link Chart.renderer}.
  * The renderer can also be used completely decoupled from a chart.
  *
+ * See [How to use the SVG Renderer](
+ * https://www.highcharts.com/docs/advanced-chart-features/boost-module) for a
```
Comment (KacperMadej) [inline ts/Core/Renderer/SVG/SVGRenderer.ts:105]: Shouldn't this be linked to the renderer docs?

**Review by KacperMadej — CHANGES_REQUESTED** [review]
The new doc page looks easy to understand 👍

**Pull request merged**